source/FreeplayState.hx

```diff
@@ -177,11 +177,11 @@ class FreeplayState extends MusicBeatState
 		add(textBG);
 
 		#if PRELOAD_ALL
-		var leText:String = "Press SPACE to listen to the Song / Press CTRL to open the Gameplay Changers Menu / Press RESET to Reset your Score and Accuracy.";
-		var size:Int = 16;
+		var leText:String = "1.1 is gonna be lit yo! also press space to listen to a song. its a good tip!";
+		var size:Int = 17;
 		#else
-		var leText:String = "Press CTRL to open the Gameplay Changers Menu / Press RESET to Reset your Score and Accuracy.";
-		var size:Int = 18;
+		var leText:String = "press space to listen to a song your hovering over. also 1.1 is gonna be lit!";
```

Comment:
you're*
![youre-your](https://user-images.githubusercontent.com/58280776/148662158-0fc64505-ee63-42e7-8d82-fc3b289128c3.gif)